Tools/scripts/decode_devid.py

```diff
@@ -129,11 +129,13 @@ def num(s):
     0x0F : "DEVTYPE_BARO_ICP101XX",
     0x10 : "DEVTYPE_BARO_ICP201XX",
     0x11 : "DEVTYPE_BARO_MS5607",
-    0x12 : "DEVTYPE_BARO_MS5837",
+    0x12 : "DEVTYPE_BARO_MS5837_30BA",
     0x13 : "DEVTYPE_BARO_MS5637",
     0x14 : "DEVTYPE_BARO_BMP390",
     0x15 : "DEVTYPE_BARO_BMP581",
     0x16 : "DEVTYPE_BARO_SPA06",
+    0x17 : "DEVTYPE_BARO_AUAV",
+    0x18 : "DEVTYPE_BARO_MS5837_02BA",
```

Comment:
Once this is merged we need to update https://github.com/ArduPilot/WebTools/blob/main/Libraries/DecodeDevID.js too.